coinstacks/bitcoin/daemon/test.sh

```diff
@@ -0,0 +1,214 @@
+#!/bin/bash
```

Comment:
Can probably remove this file. Or we should update with bitcoin addresses/rpc calls. Added to do some performance checking on geth version bumps since it seemed like there was performance degradation on one of the releases. Honestly haven't seen anything weird since then so probably not super userful